.github/workflows/knative-mkdocs.yaml

```diff
@@ -28,4 +28,4 @@ jobs:
 
       - name: Build docs 
         run: |
-          mike deploy dev --push
+          mike deploy latest --push
```

Comment:
I don't think we want to do this quite like this. The way Mike is normally used is to point latest at a particular version (once we have docs built by version) using mikes alias command, which redirects an alias to a version. That way latest will redirect to the latest release (eg v0.22, v0.23 etc).

If we want to update latest to keep pointing to dev for now for test purposes until we have actual releases built we could change the command to "mike deploy dev latest -u --push". This will build the docs as "dev", and update the latest alias to point to it. Eventually I'd expect us to have an action that runs on release (not every commit) that would do "mike deploy $TAG latest -u --push"

Reply:
Tldr let's do:

~~~~suggestion 
mike deploy dev latest -u --push
~~~~

:)

Reply:
It was more simple than that.
The feedback is no understanding what the term "dev" is? Seein "dev" on the drop-down doesn't mean anything. People don't understand that it means the last changes in the development branch call master/main.

And some folks take `latest` as the last changes in the main/master branch, not the latest released/cut versioned software.

In knative.dev we name it in the drop down "pre-release" maybe this is the best. 

For now I will make it
```
mike deploy dev latest -u --push
```

Thanks for the explanation, I'm not familiar with mike.

Reply:
Gotcha, makes sense. Yeah pre-release (`mike deploy pre-release --push`) would work too, just `latest` should be an alias rather than a direct version or we'll have problems later is all.